### ISSUE TITLE
selinux: Cockpit doesn't start with empty context in service file

### ISSUE THREAD

**Issue by dperpeet** [cockpit-project/cockpit#3187]
```
Nov 19 11:52:20 tester.nest.lan systemd[1]: Starting Cockpit Web Service...
Nov 19 11:52:20 tester.nest.lan remotectl[7684]: /usr/bin/chcon: failed to change context of ‘/etc/cockpit/ws-certs.d/~self-signed.cert’ to ‘system_u:object_r::s0’: Invalid argument
Nov 19 11:52:20 tester.nest.lan remotectl[7684]: remotectl: couldn't change SELinux type context '' for certificate: /etc/cockpit/ws-certs.d/~self-signed.cert: Child process exited with code 1
Nov 19 11:52:20 tester.nest.lan systemd[1]: cockpit.service: control process exited, code=exited status=1
Nov 19 11:52:20 tester.nest.lan systemd[1]: Failed to start Cockpit Web Service.
Nov 19 11:52:20 tester.nest.lan systemd[1]: Unit cockpit.service entered failed state.
Nov 19 11:52:20 tester.nest.lan systemd[1]: cockpit.service failed.
```

The service file has the entry
```
ExecStartPre=/usr/sbin/remotectl certificate --ensure --user=root --group= --selinux-type=
```

**Comment by gps1539** [comment]
Hi, I see the same issue on arch linux, is there a workaround ?

**Comment by stefwalter** [comment]
You can remove the ``` --selinux-type=``` from the ```cockpit.service``` file (or patch it out of the build).

**Comment by gps1539** [comment]
@stefwalter Thanks that worked

**Comment by stefwalter** [comment]
Fixed in 6246c16d0bed3ce63f3b598684849823d8661282
